### PR TITLE
Add pin to upstream pytorch 1.13

### PR DESCRIPTION
**Pull request by vanbasten23**

### CODE REVIEW

File: .torch_pin [hunk 1]
```diff
@@ -0,0 +1,2 @@
+release/1.13
+
```
Comment (JackCaoG) [inline .torch_pin:2]: remove extra line

Comment (vanbasten23) [reply .torch_pin]: done

**Review by JackCaoG — APPROVED** [review]
Do we expect CI to fail without your other pr? If that's the case you can force merge this one.

**Comment by vanbasten23** [comment]
> Do we expect CI to fail without your other pr? If that's the case you can force merge this one.

I don't expect the CI to fail.
Thanks for the quick review!

**Pull request merged**